src/Components/RelatedCollectionsRail/RelatedCollectionEntity.tsx

```diff
@@ -0,0 +1,157 @@
+import { Box, color, Flex, Link, Sans, Serif } from "@artsy/palette"
+import { RelatedCollectionEntity_collection } from "__generated__/RelatedCollectionEntity_collection.graphql"
+import { track } from "Artsy/Analytics"
+import * as Schema from "Artsy/Analytics/Schema"
+import currency from "currency.js"
+import { map } from "lodash"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { data as sd } from "sharify"
+import styled from "styled-components"
+
+export interface CollectionProps {
+  // collection: ArtistCollectionEntity_collection
+  collection: RelatedCollectionEntity_collection
+}
+
+@track()
+export class RelatedCollectionEntity extends React.Component<CollectionProps> {
+  @track<CollectionProps>(({ collection }) => ({
```

Comment:
This work is already all done so not suggesting we refactor everything :) but we can now ditch all of the classes and use our new [`useTracking()` hook](https://github.com/artsy/reaction/blob/master/src/Components/NavBar/Menus/MoreNavMenu.tsx#L7).

Reply:
Sweet!! Good to know.